src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final char SEPARATOR = ',';
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int nextIndex = 1;
 
+    public void getStatistic(String fromFileName, String toFileName) {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
```

Comment:
[Divide program logic](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#divide-program-logic) - I recommend the method readFromFile, createReport and writeToFile